runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -363,4 +397,17 @@ protected void markTile(LocalPoint localPoint)
 
 		loadPoints();
 	}
+
+	private void clearLocalMarkerPoints()
+	{
+		//Clear all points
```

Comment:
Not too sure of the value of these comments, can you remove ones which are not important. There's quite a few in this file - I would go through and review them all. (I'm not sure of the official policy of in-line documentation opposed to doc-comments)